pages/linux/udisktctl.md

```diff
@@ -0,0 +1,24 @@
+# udisktctl
+
+> A command-line program used to interact with the udisksd daemon process.
+> More information: <https://manpages.ubuntu.com/manpages/focal/man1/udisksctl.1.html>.
+
+- Show high-level information about disk drives and block devices:
+
+`udisksctl status`
+
+- Show detailed information about a device:
+
+`udisksctl info --block-device {{/dev/device_name}}`
+
+- Mount a device and prints the mount point:
+
+`udisksctl mount --block-device {{/dev/device_name}}`
+
+- Unmount a device:
+
+`udisksctl unmount --block-device {{/dev/device_name}}`
```

Comment:
Could be 1 partition or all the partitions?
```suggestion
`udisksctl unmount --block-device {{/dev/sdXN}}`
```

Reply:
Hey, thanks for mentioning this. I checked it myself. The partition path should be passed here. The same goes for the `mount`. I fixed both of them in a separate commit.

Reply:
Is `partition` better than `device` when we aren't talking about a full device?

Reply:
You're right, however, its man page used the term `device` for both mount/unmount options. But as I tested it now, it can only mount partitions. I'm not sure which term suits best here. Yet, are you suggesting to change it to `device`?

Reply:
I went with `device partition` term to be more specific about the command.